api_tests/reviews/mixins/filter_mixins.py

```diff
@@ -84,9 +84,12 @@ def user(self, allowed_providers):
         return user
 
     def test_filter_actions(self, app, url, user, expected_actions):
-        # unfiltered
-        expected = {item._id for item in expected_actions}
-        actual = get_actual(app, url, user)
+        # Strip any version suffixes from expected IDs
+        def unversion_id(_id):
```

Comment:
Similarly, we should test versioned_id works instead of trying to chop off the version.